includes/classes/admin/options-pages/styles-option-modules.php

```diff
@@ -132,6 +132,27 @@
                         'search_placeholder' => '',
                         'placeholder'        => '',
                     ),
+                    array(
+                        'key'               => 'field_tailwindcss_cdn',
+                        'label'             => __( 'Enqueue TailwindCSS CDN?', 'pilopress' ),
+                        'name'              => 'tailwindcss_cdn',
+                        'type'              => 'true_false',
+                        'instructions'      => __( 'Load play.tailwind.css CDN for easier development. Untoggle when you compiled and are in prod environnement. By default latest version of CDN is loaded, you can change it with the filter `pip/tailwind/cdn_version`.', 'pilopress' ),
```

Comment:
I would also add _(minimum version requirement: 3.x.x because you can't use it with v2 for example)_